lib/mpi/service.rb

```diff
@@ -59,8 +59,10 @@ def add_person(user_identity)
     # @param user [UserIdentity] the user to query MVI for
     # @return [MPI::Responses::FindProfileResponse] the parsed response from MVI.
     # rubocop:disable Metrics/MethodLength
-    def find_profile(user_identity, search_type = MPI::Constants::CORRELATION_WITH_RELATIONSHIP_DATA)
-      profile_message = create_profile_message(user_identity, search_type: search_type)
+    def find_profile(user_identity,
+                     search_type = MPI::Constants::CORRELATION_WITH_RELATIONSHIP_DATA,
```

Comment:
I wonder if it'd be better to make all of these named params, so we're not mixing both types?
`def find_profile(user_identity: user_identity, search_type: MPI::Constants..., orch_search: false)`